### ISSUE TITLE
Issue: faucet

### ISSUE THREAD

**Issue by mochet** [Joystream/marketing#19]
The current faucet in Discord/Atlas appears to have some issues and I am just making this issue to ensure that we have some way of dealing with this (I guess it is part of the onboarding process?)

Currently users are given `101 tJOY` with is not a decent amount. This means we are seeing the facuet on Discord being sybil attacked (the exact reason why people are sybil attacking it is unknown, because it gives out such a low amount it is of no possibly financial benefit to try and manipulate)

**Comment by bedeho** [comment]
Can't greeters/HR be the people doing this? Why isn't 101 enough? enough for what?

**Comment by mochet** [comment]
> Can't greeters/HR be the people doing this? Why isn't 101 enough? enough for what?

Yes, so long as there is some plan with how to deal with this. At scale, the system we currently have doesn't really work.

101 tokens
* only only allows for membership + forum posts + video/channel creation (the latter items not costing anything at the moment)
* doesn't allow a user to create an election announcement, nor create any proposals.

Beyond that, If in `olympia` costs for uploading, posting on the forum and other things, the amount will need to be adjusted.
Presumably, with the membership group being added there will also be implications--I am not sure exactly what these are, so it would be (i.e. will membership become more expensive).
